modules/change_management/mmd.json

```diff
@@ -9,7 +9,7 @@
     "queueable": true,
     "trackable": true,
     "taggable": true,
-    "peerReplicable": true,
+    "peerReplicable": false,
```

Comment:
The "peerReplicable" should be set as `true`

Reply:
Why we should need to set to true?